### ISSUE TITLE
Disabling default hotkey binding doesn't get saved

### ISSUE THREAD

**Issue by vadosnaprimer** [clementgallet/libTAS#499]
https://www.youtube.com/watch?v=M8oiJhoXHnw

Clearing a default binding, even with the executional selected, doesn't survive the restart. Assigning it to a different key works tho.

**Comment by clementgallet** [comment]
Indeed. Because of how I store hotkeys, there is no distinction between disabled hotkey and no hotkey. Because hotkeys are regularly being added, I set to default hotkey when no hotkey is set in the config file.

Not sure what is the best option yet.
